src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -619,6 +620,13 @@ export class EditDataEditor extends BaseEditor {
 			} else {
 				this._sash.hide();
 			}
+			if (this._resultsEditorContainer) {
+				if (this.queryPaneEnabled()) {
+					DOM.toggleClass(this._resultsEditorContainer, 'tabbedPanel', true);
```

Comment:
What made you choose tabbedPanel for this? In general unless this is actually a tabbed panel we shouldn't be taking CSS from random other parts of the application - it makes it harder to determine how changing styles will affect the look and feel if things are pulling in random dependencies like this.

Reply:
originally it was to use the border style that was provided by it, but since @alanrenmsft changed it to a new format, I will need to adapt it accordingly